tests/queries/1_stateful/00174_distinct_in_order.sql

```diff
@@ -0,0 +1,10 @@
+select '-- check that distinct with and w/o optimization produce the same result';
```

Comment:
Also consider to add setting `optimize_distinct_in_order` to https://github.com/ClickHouse/ClickHouse/blob/8e98b7d47dbbcfe88c8d420b35794854e93a6361/tests/clickhouse-test#L398-L450.

Reply:
done